templates/6_tma_batch_qc_metrics.ipynb

```diff
@@ -0,0 +1,546 @@
+{
```

Comment:
<p>Not having these names match up with the example plots is confusing. I think it makes sense to create fake names for the plot that will match up with the generic names here. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/toffy/pull/358/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='38'/>